app/views/events/_upcoming_events.html.erb

```diff
@@ -0,0 +1,31 @@
+<section>
+  <div class="container marketing">
+    <div class="row">
+        <div class="col-lg-4">
```

Comment:
I think col-lg-4 will result in these single column on anything smaller than a large screen. I think col-md-4 would be nicer, so it remains with three events on medium screens before collapsing when smaller than that.